screens/Landmark.js

```diff
@@ -157,7 +162,8 @@ class LandmarkScreen extends React.Component {
 
 const mapDispatchToProps = dispatch => ({
   fetchDirections: coordinate => dispatch(getDirections(coordinate)),
-  eraseDirections: () => dispatch(clearDirections())
+  eraseDirections: () => dispatch(clearDirections()),
+  clearLandmark: () => dispatch(clearLandmark())
```

Comment:
Can we change this to eraseLandmark? I think it'll be easier to read/debug if need be.

Reply:
i want to avoid using different words that kinda mean the same thing, because i think it can be confusing, but i agree re: reading/debuging. what if we do something like `clearLandmark: () => dispatch(clearLandmarkAction())`

Reply:
Sounds good to me! Can you  make this change for both clearLandmark and clearDirections? That sets a nice pattern for us go forward :)